content/en/Platform Deep Dive/Cobalt Account/password-best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Password Best Practices"
+linkTitle: "Password Best Practices"
+weight: 40
+description: >
+  Learn how to create a strong password.
+---
+
+{{% pageinfo %}}
+Follow best practices when creating a password for your Cobalt account. A strong password helps you protect your account from unauthorized access.
+{{% /pageinfo %}}
+
+## Create a Strong Password
+
+**Cobalt password requirements**:
+
+- The password must contain:
+  - At least 10 characters
+  - At least 3 of the following:
+  - Lowercase letters: a–z
+  - Uppercase letters: A–Z
+  - Numbers: 0–9
+  - Special characters: !@#$%^&*
+- We don't allow common passwords such as `password`, `123456`, or `qwerty`.
+
+Example of a strong password: `1$WinterIsComing$1` (don't use this example).
+
+**How to create a strong password**:
+
+- Use a unique passphrase that you can memorize but no one else can guess, such as an abbreviation made from the first letters of your favorite quote.
+- Use a long passphrase that you can memorize. The longer your password is, the harder it is for an attacker to compromise it.
+- If you find it difficult to remember multiple unique passwords, use a trusted password management tool.
```

Comment:
This question is still pending.

Reply:
Cullen has no concerns with saying this. we can proceed. https://cobalthq.slack.com/archives/D03F95DGH1R/p1685048653507299